File: Java/Tarjeta.java
package Java;

class Tarjeta extends Payment {
    Integer number;
    Integer cvv;
    Integer date;
    String name;
    public Tarjeta (String name, Integer number, Integer cvv, Integer date, Integer id){
        super(id);
        this.name = name;
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}

File: Python/tarjeta.py
from payment import Payment

class Tarjeta(Payment):
    name = str
    number = int
    cvv = int
    date = int
    def __init__(self, id, name, number, cvv, date):
        super().__init__(id)
        self.name = name
        self.number = number 
        self.cvv = cvv
        self.date = date